src/explore-education-statistics-frontend/src/services/googleAnalyticsService.ts

```diff
@@ -55,15 +55,23 @@ export function logEvent({ category, action, label, value }: AnalyticsEvent) {
   });
 }
 
-export function logOutboundLink(label: string, url: string) {
-  if (!initialised) {
+export function logOutboundLink(label: string, url: string, newTab?: boolean) {
+  if (
+    process.env.DATA_API_BASE_URL !== 'http://localhost:5000/api' &&
+    !initialised
+  ) {
     return;
   }
+
   ReactGA.outboundLink(
     {
       label,
     },
+    // eslint-disable-next-line consistent-return
     function hitCallback() {
+      if (newTab) {
+        return window.open(url, '_blank');
```

Comment:
If you put the return after `window.open...`, so it's not returning anything just preventing it continuing through the code, then shouldn't need to disable consistent-return.